Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/CompilationErrors.dfy

```diff
@@ -1,8 +1,8 @@
-// RUN: %exits-with 3 %dafny /allowAxioms:0 "%s" > "%t"
+// RUN: %exits-with 3 %build "%s" > "%t"
 // RUN: %diff "%s.expect" "%t"
 
-type MyType  // compile error: abstract type
```

Comment:
I'm confused about this change. It should still be a compile error, shouldn't it?

Reply:
I don't know. The type is not used by the rest of the code, maybe that matters.